libc-bottom-half/cloudlibc/src/libc/dirent/readdir.c

```diff
@@ -38,6 +41,8 @@ static_assert(DT_UNKNOWN == __WASI_FILETYPE_UNKNOWN, "Value mismatch");
   } while (0)
 
 struct dirent *readdir(DIR *dirp) {
+  struct stat statbuf;
```

Comment:
[minor] the scope of this variable could be reduced to the block starting in [L#93](https://github.com/WebAssembly/wasi-libc/pull/345/files#diff-39125f185a8b15746cc411f8d6fe6f79838f07c3bd95b773b12644a62da4eb80R93)

Reply:
Fixed.